src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/gcc.stack_protector.a.sarif

```diff
@@ -42,12 +42,13 @@
       "artifacts": [
         {
           "location": {
-            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/gcc.stack_protector.a"
+            "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/gcc.stack_protector.a",
+            "index": 0
           },
           "hashes": {
-            "md5": "764B913022B393EAF22F96014401576F",
-            "sha-1": "4B49DA507C9212490EE09280A5A7AD1F1A389188",
-            "sha-256": "8548292CA34C14D090A1DE7327874010E3A7ACE5846B2E28D743CC0F35C069CB"
+            "md5": "B3AA4DD82C145C4B6E589953A2A2FAC4",

```

Comment:
>B3AA4DD82C145C4B6E589953A2A2FAC4 [](http://example.com/codeflow?start=20&length=32)

Another one, Shaopeng. I'm guessing we recompiled these binaries. What this means is that changes in scan target hashes must not break our functional tests when that occurs?

Reply:
I'm guessing we recompiled these binaries. ---- no, this is a new bug in SDK, I will resolve this comment, please see my comment on the other.